microsetta_public_api/api/microsetta_public_api.yml

```diff
@@ -36,6 +36,62 @@ paths:
         '404':
           $ref: '#/components/responses/404SampleNotFound'
 
+  '/diversity/alpha_group/{alpha_metric}':
+    post:
+      operationId: microsetta_public_api.api.diversity.alpha.alpha_group
+      tags:
+        - Diversity
+      summary: Query alpha diversity for a group of samples
+      description: Query alpha diversity for a group of samples
+
+      parameters:
+        - $ref: '#/components/parameters/alpha_metric'
+
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - sample_ids
+              properties:
+                sample_ids:
+                  type: array
```

Comment:
ya I _think_ it should come out of flask as a `list`...

Reply:
yup.